src/include/access/k2/pg_session.h

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
I think this should be covered by the k2 txn manager, no?

Reply:
This is the session object related to each database client connection, for example, we need to keep track of the current database for this session. This object could be integrated with k2 txn manager later.